lib/mayaUsd/CMakeLists.txt

```diff
@@ -29,9 +29,17 @@ endif()
 find_package(Boost COMPONENTS filesystem REQUIRED)
 
 # -----------------------------------------------------------------------------
-# compile definitions
+# compiler configuration
 # -----------------------------------------------------------------------------
-target_compile_definitions(${PROJECT_NAME} 
+target_compile_definitions(${PROJECT_NAME}
+    PUBLIC
+        USD_VERSION_NUM=${USD_VERSION_NUM}
```

Comment:
This seems ok for now, but since it comes from a dependency, maybe it should be on the upstream libraries to ensure this is set on the imported target with `INTERFACE_COMPILE_DEFINITIONS`?